code/edgeidentity/src/androidTest/java/com/adobe/marketing/mobile/edge/identity/IdentityECIDHandlingTest.java

```diff
@@ -12,13 +12,12 @@
 package com.adobe.marketing.mobile.edge.identity;
 
 import static com.adobe.marketing.mobile.edge.identity.IdentityFunctionalTestUtil.*;
```

Comment:
We should follow some pattern in all the test classes, either we do * import or just import required functions/fields